mmaction/models/heads/lfb_head.py

```diff
@@ -0,0 +1,122 @@
+import warnings
+
+import numpy as np
+
+from mmaction.core.bbox import bbox2result
+
+try:
+    import mmdet  # noqa
+    from mmdet.core.bbox import bbox2roi
+    from mmdet.models import HEADS as MMDET_HEADS
+    from mmdet.models.roi_heads import StandardRoIHead
+except (ImportError, ModuleNotFoundError):
+    warnings.warn('Please install mmdet to use bbox2roi, MMDET_HEADS '
+                  'and StandardRoIHead')
+
+if 'mmdet' in dir():
```

Comment:
Better use the new checking style (after successfully importing all modules from mmdet, assign mmdet_imported as True). Cuz sometimes, you may successfully import mmdet, but not the submodules.